packages/SwingSet/test/gc/test-gc-vat.js

```diff
@@ -134,8 +134,8 @@ test('forward to fake zoe', async t => {
   console.log(`targetID: ${targetID}`);
 
   // confirm that zoe is exporting it
-  t.is(findClist(c, zoeID, invitation), 'o+9');
-  t.true(dumpClist(c).includes(`${invitation}/${zoeID}/o+9`));
+  t.is(findClist(c, zoeID, invitation), 'o+10'); //
```

Comment:
Dangling empty comment?

Reply:
oops, yes, that was my reminder that this line was failing, before I updated the o+9 to o+10. Good catch, I'll remove it.